language/tokens.ts

```diff
@@ -147,6 +147,17 @@ const commonMatchers: Matcher[] = [
       type: `word`
     }
   },
+  {
+    name: `WHEN-GROUP`,
+    match: [
+      { type: `word`, match: (word) => word.toUpperCase() === `WHEN` },
```

Comment:
Only the `WHEN`-part is marked when linter detects errors:

![image](https://github.com/codefori/vscode-rpgle/assets/13275072/ec45c1e3-29b0-442f-8956-afbacf248664)

And not-existing keywords are accepted (`WHEN-DONE`)...

Could you add the valid second part of the `WHEN` keyword? Like the `ON-ERROR`and ´ON-EXCP`?